harness/determined/common/api/authentication.py

```diff
@@ -125,8 +125,6 @@ def _init_session(
         fallback_to_default = password is None and session_user == constants.DEFAULT_DETERMINED_USER
         if fallback_to_default:
```

Comment:
rb: unsure if there's a better implementation that's obviously correct (also, is there a codepath where this isn't correct)

Reply:
Likely to be only piecemeal improvements. I'll list them here when they occur to me:
* `default_load_user_password` should probably prefer `constants.DEFAULT_DETERMINED_USER` / `constants.DEFAULT_DETERMINED_PASSWORD` over `None` / `None`.